src/main/resources/com/google/api/codegen/java/message.snip

```diff
@@ -112,7 +112,7 @@
   }
   private static final {@schema.innerTypeName} DEFAULT_INSTANCE;
   static {
-    DEFAULT_INSTANCE = new {@schema.innerTypeName}();
+      DEFAULT_INSTANCE = new {@schema.innerTypeName}();
```

Comment:
I'm not sure how else to make the baseline indentation look nice. With just one tab, the resulting view is
```
static {
DEFAULT_INSTANCE
}
```

but the two tab indentation of the latest revision results in
```
static {
  DEFAULT_INSTANCE
}
```

My guess is that the curly braces mess with the snippet parsing whitespacing.